bundle/config/mutator/rewrite_sync_paths.go

```diff
@@ -49,11 +49,27 @@ func (m *rewriteSyncPaths) Apply(ctx context.Context, b *bundle.Bundle) diag.Dia
 			if err != nil {
 				return dyn.InvalidValue, err
 			}
-			v, err = dyn.Map(v, "include", dyn.Foreach(m.makeRelativeTo(b.BundleRootPath)))
+
+			// Convert include and exclude in the sync block to use Unix-style slashes.
+			// This is required for the ignore.GitIgnore we use in libs/fileset to work correctly.
+			v, err = dyn.Map(v, "include", dyn.Foreach(func(p dyn.Path, val dyn.Value) (dyn.Value, error) {
+				relPath, err := m.makeRelativeTo(b.BundleRootPath)(p, val)
+				if err != nil {
+					return dyn.InvalidValue, err
+				}
+				return dyn.NewValue(filepath.ToSlash(relPath.MustString()), relPath.Locations()), nil
```

Comment:
Let's not panic with MustString but skip it instead if not a string.

Reply:
`makeRelativeTo` calls MustString on the same value above so it's safe to use it, any concerns with it?

Reply:
Yes, let's not call Must* on any of the values. There is no benefit but there is always a risk of panic. Even if if today it's safe, it requires thorough understanding of the code to prove that it is and it also might stop being safe later.

Reply:
There's a semantical difference between using both, MustString implies it has to be the string and if it's not something is certainly wrong hence the panic while AsString implies that it might be or not be string. In this case it has to be string. With using AsString it would still error out and prevent further execution of the command but I'm fine not making it panic